pkgs/applications/science/medicine/aliza/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, itk
+, cmake
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aliza";
+  version = "1.9.7";
+
+  src = fetchFromGitHub {
+    owner = "AlizaMedicalImaging";
+    repo = "AlizaMS";
+    rev = "v${version}";
+    hash = "sha256-bambl3NP42KEwh6NPb8jhQlGqaUCSt6QuzUmmBnEAQw=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    itk
+    cmake
+    qtbase
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE:STRING=Release"
+    "-DALIZA_QT_VERSION:STRING=5"
+    "-DITK_DIR:STRING=${itk}"
+    "-DMDCM_USE_SYSTEM_ZLIB:BOOL=ON"
```

Comment:
```suggestion
    (lib.cmakeFeature "CMAKE_BUILD_TYPE" "Release")
    (lib.cmakeFeature "ALIZA_QT_VERSION" "5")
    (lib.cmakeOptionType "filepath" "ITK_DIR" itk)
    (lib.cmakeBool "-MDCM_USE_SYSTEM_ZLIB" true)
```

Reply:
are you sure `(lib.cmakeBool "-MDCM_USE_SYSTEM_ZLIB" true)` is correct?

Reply:
... why not?